daemon/graphdriver/overlay/overlay.go

```diff
@@ -338,7 +339,7 @@ func (d *Driver) Remove(id string) error {
 }
 
 // Get creates and mounts the required file system for the given id and returns the mount path.
-func (d *Driver) Get(id string, mountLabel string) (string, error) {
+func (d *Driver) Get(id string, mountLabel string) (_ string, retErr error) {
```

Comment:
I've never seen this done before.  `(_ string)`.  I don't know if this is cool or something horrid